Scripts~/windows/.clean-exports.ps1

```diff
@@ -1,22 +1,52 @@
 $ROOT_DIR="."
```

Comment:
Should we use "$PSScriptRoot" instead of "."? We probably don't want to pollute whatever directory the user is in.

Reply:
I'm using the `build` folder now to dump all build/release artifacts that should not be included in the tarball export process (aka dlls and the sample package should still be in path as previously but the zips and tarballs will no longer be in the root directory)